frontend/src/views/MovieDetailsScreen.vue

```diff
@@ -42,7 +42,7 @@ export default {
 
     async fetchMovie (id) {
       try {
-        const { data: movie } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movies-${id}`, {
+        const { data: movie } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movies-${id}/`, {
```

Comment:
On the screenshots there are red requests, it means they failed for some reason(in your case 404 error, it means endpoint does not exist, thus you should fix it). And for this reason movie images are not shown as well.
Review please all endpoints front-end uses, check if they work and fix them